README.md

```diff
@@ -25,6 +25,7 @@ bpftrace is a high-level tracing language for Linux. bpftrace uses LLVM as a bac
 - [Probe types](#probe-types)
 - [Plugins](#plugins)
 - [License](#license)
+- [Code of Conduct](docs/code_of_conduct.md)
```

Comment:
Why is the code of conduct in `docs` but the contributing guide is in the base directory? Both are acceptable, but I'm wondering why they are different: https://github.com/joelparkerhenderson/github-special-files-and-paths?tab=readme-ov-file#contributing

I would personally recommend just putting all the "special" files in the root, but I think it should be consistent at least.